src/iexamine.cpp

```diff
@@ -56,39 +56,34 @@ void iexamine::gaspump(player *p, map *m, int examx, int examy)
     add_msg(m_info, _("Out of order."));
 }
 
-void iexamine::atm(player *p, map *m, int examx, int examy)
+void iexamine::atm(player *const p, map *, int , int )
 {
-    (void)m; //unused
```

Comment:
question: does one have to declare unused arguments as void when left out of prototype and function decl?